RUFAS/data_structures/manure_stream.py

```diff
@@ -0,0 +1,103 @@
+from typing import Optional
+
+from RUFAS.data_structures.pen_manure_data import PenManureData
+
+
+class ManureStream:
+    """
+    This class packages manure data.
+
+    Parameters
+    ----------
+    water : float
+        Mass of water in the manure stream (kg).
+    ammoniacal_nitrogen : float
+        Mass of ammoniacal nitrogen in the manure stream (kg).
+    nitrogen : float
+        Mass of total nitrogen in the manure stream (kg).
+    phosphorus: float
+        Mass of phosphorus in the manure stream (kg).
+    potassium : float
+        Mass of potassium in the manure stream (kg).
+    ash : float
+        Mass of ash in the manure stream (kg).
+    non_degradable_volatile_solids : float
+        Mass of non-degradable volatile solids in the manure stream (kg).
+    degradable_volatile_solids : float
+        Mass of degradable volatile solids in the manure stream (kg).
+    total_solids : float
+        Mass of total solids in the manure stream (kg).
+    volume : float
+        Volume of the manure stream (m^3).
+    pen_manure_data : PenManureData | None
+       Optional, more specific information about the manure and the pen or pens that produced it.
+
+    Attributes
+    ----------
+    water : float
+        Mass of water in the manure stream (kg).
+    ammoniacal_nitrogen : float
+        Mass of ammoniacal nitrogen in the manure stream (kg).
+    nitrogen : float
+        Mass of total nitrogen in the manure stream (kg).
+    phosphorus: float
+        Mass of phosphorus in the manure stream (kg).
+    potassium : float
+        Mass of potassium in the manure stream (kg).
+    ash : float
+        Mass of ash in the manure stream (kg).
+    non_degradable_volatile_solids : float
+        Mass of non-degradable volatile solids in the manure stream (kg).
+    degradable_volatile_solids : float
+        Mass of degradable volatile solids in the manure stream (kg).
+    total_solids : float
+        Mass of total solids in the manure stream (kg).
+    volume : float
+        Volume of the manure stream (m^3).
+    pen_manure_data : PenManureData | None
+       Optional, more specific information about the manure and the pen or pens that produced it.
+
+    """
+
+    def __init__(
+        self,
+        water: float,
+        ammoniacal_nitrogen: float,
+        nitrogen: float,
+        phosphorus: float,
+        potassium: float,
+        ash: float,
+        non_degradable_volatile_solids: float,
+        degradable_volatile_solids: float,
+        total_solids: float,
+        volume: float,
+        pen_manure_data: Optional[PenManureData],
```

Comment:
Should this match the docstring?
```suggestion
        pen_manure_data : PenManureData | None,
```